spyder/widgets/collectionseditor.py

```diff
@@ -916,6 +924,10 @@ def mouseDoubleClickEvent(self, event):
         else:
             event.accept()
 
+    def mouseMoveEvent(self, event):
+        """Change cursor shape."""
+        self.setCursor(Qt.PointingHandCursor)
```

Comment:
I really like this new interaction!! Great work @jsbautista!

However, I forgot to mention it to @conradolandia (sorry about that). Will do in our next UX/UI meeting and let you know what he thinks.

Reply:
I talked to @conradolandia and @CAM-Gerlach about this in our last UX meeting and they also it's a great way to let users know that entries in the Variable Explorer are clickable.

Our only suggestion is to change the cursor shape to `PointingHandCursor` only when the cursor is on top of a variable row. Right now, this changes the shape as soon as the cursor is on top of the Variable Explorer.